views/get-started.erb

```diff
@@ -25,35 +25,65 @@
 		<div class="column-two-thirds">
 			<h1 class="heading-xlarge">Get started with GOV.UK PaaS</h1>
 			<h2 class="heading-large">Evaluate GOV.UK PaaS</h2>
+			<h3 class="heading-medium">Check our features and pricing</h3>
 			<p>
-				If you’re considering cloud-based solutions for your web app, GOV.UK PaaS offers a trial period with a set of basic features and a capped <a href="https://docs.cloud.service.gov.uk/managing_apps.html#quotas">quota</a> of compute and storage. Keep in mind that you can’t use the trial version of the platform for production or live services.
+				Make sure we currently support the <a href="https://docs.cloud.service.gov.uk/before_you_start.html#before-you-start">language, framework</a> and <a href="/features">backing services</a> you need. Our <a href="/roadmap">roadmap</a> lists the features planned for release in the coming months.
```

Comment:
`/features` gives info on languages/frameworks supported, so we could only link to the one place.